config/prometheus-collectors.php

```diff
@@ -12,7 +12,7 @@
     */
 
     'queues' => [
```

Comment:
Can you now leave a comment about which way around the key => values are for labelling. Then I think it is ready